### PR TITLE
Infracost

### PR DESCRIPTION
**Pull request by 0katrinpetrosyan0**

### CODE REVIEW

File: .github/workflows/infracost.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+on: [pull_request]
+env:
+    SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+jobs:
+  infracost:
+    name: Infracost
+    runs-on: ubuntu-latest
+    env:
+      TF_ROOT: momodules/
```
Comment (mrdntgrn) [inline .github/workflows/infracost.yaml:9]: seems some misspelling ? momodules => modules

File: .github/workflows/infracost.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+on: [pull_request]
+env:
+    SSH_AUTH_SOCK: /tmp/ssh_agent.sock
```
Comment (mrdntgrn) [inline .github/workflows/infracost.yaml:3]: is this something that you use?
does this file exist inside runners?

File: .github/workflows/infracost.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+on: [pull_request]
+jobs:
+  infracost:
+    name: Infracost
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout PR branch
```
Comment (mrdntgrn) [inline .github/workflows/infracost.yaml:7]: why we need this step, it seems we can go without this?

File: .github/workflows/infracost.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      - name: Checkout base branch
+        uses: actions/checkout@v2
+        with:
+          ref: '${{ github.event.pull_request.base.ref }}'
+          fetch-depth: 0
+
+      - name: Infracost Breakdown
+        run: |
+          for i in modules/*/ ; do
+            infracost breakdown --path ${i} --format=json --out-file=${i}.json
```
[the remark this answers is not in the document]
Comment (0katrinpetrosyan0) [reply .github/workflows/infracost.yaml:26]: yes we switched to base ref, so what i need ? not to switch? but it is calculating PR based cost

**Pull request closed without merging**